package.json

```diff
@@ -65,7 +65,7 @@
     "del": "^3.0.0",
     "eslint": "^4.7.2",
     "grommet-toolbox": "^2.0.0",
-    "gulp": "^3.9.1",
+    "gulp": "^3.9.0",
```

Comment:
this will install `3.9.1` right ? since we have the `^` there.

Reply:
Yes, but thus version is failed to load the babel module

Reply:
yes I get it. but this PR does not fix this problem since it will install 3.9.1 still right ?

Reply:
can you get me more details on how to reproduce this issue?